code/game/objects/items/documents.dm

```diff
@@ -115,3 +221,218 @@
+	var/message = "You turn the screw-like dial, setting the device's scanning speed to "
+	if(print_time == 0.1 SECONDS)
+		power_per_print = INSPECTOR_POWER_USAGE_NORMAL
+		print_time = 5 SECONDS
+		message += "SLOW."
+	else if(print_time == 5 SECONDS)
+		print_time = 1 SECONDS
+		message += "LIGHTNING FAST."
+	else
+		print_time = 0.1 SECONDS
+		power_per_print = INSPECTOR_POWER_USAGE_HONK
+		message += "HONK!"
+	balloon_alert(user, message)
+
+/**
+ * Reports printed by fake N-spect scanner
+ *
+ * Not valid for the bounty.
+ */
+/obj/item/paper/fake_report
+	name = "encrypted station inspection"
+	desc = "Contains no information about the station's current status."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "slip"
+	///What area the inspector scanned when the report was made. Used to generate the examine text of the report
+	var/area/scanned_area
+	show_written_words = FALSE
+
+/obj/item/paper/fake_report/proc/generate_report(area/scan_area)
+	scanned_area = scan_area
+	icon_state = "slipfull"
+
+	var/list/characters = list()
+	characters += GLOB.alphabet
+	characters += GLOB.alphabet_upper
+	characters += GLOB.numerals
+
+	var/length = rand(23, 123)
+	var/i
+	for(i = 0; i<length; i++)
+		if(prob(90))
+			info += pick_list_replacements(CLOWN_NONSENSE_FILE, "honk")
+		else if(prob(1))
+			info += pick_list_replacements(CLOWN_NONSENSE_FILE, "rare")
+		else
+			info += pick_list_replacements(CLOWN_NONSENSE_FILE, "non-honk-clown-words")
+
+/obj/item/paper/fake_report/examine(mob/user)
+	. = ..()
+	if(scanned_area?.name)
+		. += "<span class='notice'>\The [src] contains no data on [scanned_area.name].</span>"
+	else if(scanned_area)
+		. += "<span class='notice'>\The [src] contains no data on a vague area on station, you should throw it away.</span>"
+	else if(info)
+		icon_state = "slipfull"
+		. += "<span class='notice'>Wait a minute, this isn't an encrypted inspection report! You should throw it away.</span>"
+	else
+		. += "<span class='notice'>Wait a minute, this thing's blank! You should throw it away.</span>"
+
+/**
+ * # Fake report made of water
+ *
+ * Fake report but it turns into water under certain circumstances.
+ *
+ * If someone who isn't an origami master tries to fold it into a paper plane, it will make the floor it's on wet and disappear.
+ * If it is ground, it will turn into 5u water.
+ */
+/obj/item/paper/fake_report/water
+	grind_results = list(/datum/reagent/water = 5)
+
+/obj/item/paper/fake_report/water/AltClick(mob/living/user, obj/item/I)
+	if(!user.canUseTopic(src, BE_CLOSE, NO_DEXTERITY, FALSE, TRUE))
+		return
+	var/datum/action/innate/origami/origami_action = locate() in user.actions
+	if(origami_action?.active) //Origami masters can fold water
+		to_chat(user, "<span class='notice'>You fold [src] into the shape of a plane!</span>")
+		user.temporarilyRemoveItemFromInventory(src)
+		I = new /obj/item/paperplane/syndicate(loc, src)
+		if(user.Adjacent(I))
+			user.put_in_hands(I)
```

Comment:
This looks like repeated code. If so, could you put it into its own proc so we don't have two versions of the same, just two separate calls to it?

Reply:
Fixed.